packages/cli/package.json

```diff
@@ -32,15 +33,17 @@
     "@coveo/vue-cli-plugin-typescript": "^0.7.0",
     "@oclif/dev-cli": "^1.26.0",
     "@oclif/test": "^1",
+    "@types/dedent": "^0.7.0",
     "@types/fs-extra": "^9.0.6",
+    "@types/jest": "^26.0.22",
     "@types/node": "^10",
     "chai": "^4.2.0",
     "jest": "^26.6.3",
     "patch-package": "^6.2.2",
     "rimraf": "^3.0.2",
     "ts-jest": "^26.5.1",
     "ts-node": "^8",
-    "typescript": "^3.3"
+    "typescript": "4.2.3"
```

Comment:
Could be reverted:
I wanted to migrate the tsconfig to a composite project to split the test tsconfig from the source tsconfig (to avoid having describe everywhere), but was unable because of https://github.com/oclif/oclif/issues/299.
Oclif went hay-wire when I tried to split it.